<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamForm</class>
 <widget class="QWidget" name="ParamForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>266</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,1,0">
     <item row="0" column="1">
      <widget class="QSpinBox" name="criteriaCount"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListView" name="criteriaView">
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelCriteria">
       <property name="text">
        <string>Критерии</string>
       </property>
       <property name="buddy">
        <cstring>criteriaCount</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelAlternatives">
       <property name="text">
        <string>Альтернативы</string>
       </property>
       <property name="buddy">
        <cstring>alternativesCount</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="alternativesCount"/>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QListView" name="alternativesView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
